packages/destination-actions/src/destinations/the-trade-desk-crm/index.ts

```diff
@@ -1,6 +1,7 @@
 import type { AudienceDestinationDefinition, ModifiedResponse } from '@segment/actions-core'
 import type { Settings, AudienceSettings } from './generated-types'
 import { IntegrationError } from '@segment/actions-core'
+import { getAllDataSegments } from './functions'
 
 import syncAudience from './syncAudience'
 const API_VERSION = 'v3'
```

Comment:
```suggestion
export const API_VERSION = 'v3'
```

So you can reuse in tests